libs/platform/ebpf_object.c

```diff
@@ -133,6 +133,19 @@ ebpf_object_tracking_terminate()
     }
 }
 
+/**
+ * @brief Free invoked when the current epoch ends. When the object ref-count reaches zero.
```

Comment:
"When the object ref-count reaches zero" is not a complete sentence.  Not sure what was intended.